target_snowflake/__init__.py

```diff
@@ -393,27 +396,43 @@ def flush_streams(
     # Return with state message with flushed positions
     return flushed_state
 
+def _verify_snowpipe_usage(config):
```

Comment:
it's weird that we are passing config through this like we are mutating it. can it just be a call without a return, like `_ = self._verify_snowpipe_usage(config)`? also can you add typing here, leave it better than we found it n such?